### PR TITLE
Add https_only setting for app service - closes #101

### LINKED ISSUE #101: https_only missing for app service

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

File: src/Farmer/Builders.WebApp.fs [hunk 1]
```diff
@@ -497,10 +501,12 @@ module Converters =
             location = webApp.Location.Value
             dependsOn = webApp.Dependencies |> List.map(fun p -> p.Value)
             kind = webApp.Kind
+            https_only = webApp.HTTPSOnly
```
Comment (isaacabraham) [inline src/Farmer/Builders.WebApp.fs:504]: This can be deleted.

File: src/Farmer/Builders.WebApp.fs [hunk 2]
```diff
@@ -497,10 +501,12 @@ module Converters =
...
             properties =
                 {| serverFarmId = webApp.ServerFarm.Value
                    siteConfig =
                         [ "alwaysOn", box webApp.AlwaysOn
+                          "httpsOnly", box webApp.HTTPSOnly
```
Comment (isaacabraham) [inline src/Farmer/Builders.WebApp.fs:509]: This field needs to be brought up a level, parallel with `serverFarmId` and `siteConfig` (according to [here](https://docs.microsoft.com/en-us/azure/templates/microsoft.web/sites) anyway). Also the `box` won't be necessary as the field is a boolean.

**Comment by isaacabraham** [comment]
Looks good - just a couple of tweaks needed for the outputter. Thanks :-)

**Comment by forki** [comment]
done

**Comment by isaacabraham** [comment]
Thank you :-)

**Pull request merged**